config.h

```diff
@@ -80,7 +80,6 @@
 #define HOMING_CYCLE_0 (1<<Z_AXIS)                // REQUIRED: First move 
```

Comment:
Are these homing cycles used at all? Something looks wrong if we do.  Also, the "NOTE:" at the end of the comment is wrong.